knowledge-base/visual-studio-preview-is-not-working-when-using-objectdatasource.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Report Designer Preview is not working when using ObjectDataSource
+description: Preview the report in the Visual Studio Designer is not working when using ObjectDataSource.
+type: troubleshooting
+page_title: Reports using ObjectDataSource cannot be opened with the Visual Studio Designer
+slug: visual-studio-preview-is-not-working-when-using-objectdatasource
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+    <tr>
+		<td>Report Designer</td>
+		<td>Visual Studio Designer</td>
+	</tr>
+</table>
+
+## Description
+
+Generally, the Visual Studio Report Designer operates *inside* the Visual Studio application. Thus, when previewing a report the report constructor is executed in the context of the Visual Studio application and all the configuration settings are obtained from the Visual Studio's **devenv.exe.config** file. If not taken upon consideration, this might lead to two major issues - *ConnectionStrings* and *AssemblyReferences* mismatch.   
```

Comment:
Remove "Generally" and avoid using words that don't bring value to the main thought. These are called filler words, which might be good idea in in-person communication (e.g. support tickets), but are not welcomed in documentation, nor informative articles as a whole. 

Further, the first sentence doesn't bring any value or information. Instead of stating that the VS Report Designer works inside VS, try to use this as reason for the statement in your second sentence. For example: "As the Visual Studio Report Designer operates *inside* Visual Studio, when previewing a report there, the report constructor is executed in the context of the Visual Studio application and all the configuration settings are obtained from the Visual Studio's **devenv.exe.config** file."